src/motor_hardware.cc

```diff
@@ -285,15 +286,18 @@ void MotorHardware::readInputs(uint32_t index) {
                     } else {
                         ROS_INFO_ONCE("Firmware version %d", mm.getData());
                         firmware_version = mm.getData();
-			motor_diag_.firmware_version = firmware_version;
+                        motor_diag_.firmware_version = firmware_version;
                     }
+                    publishFirmwareInfo();
```

Comment:
This only posts one time but oled does get the info.  LEAVE it for now and at some later time I will do the part that requires export of motor_hardware api over to motor_node.cc for periodic firmware publish.   Don't mess with it, lets just push this puppy.

Reply:
Yep, I've had it publish every message at first but it was just way too much spam. So I made it latched instead and every one will get the message once it subscribes to the topic. It does actually seem to get published 3 times (first twice with only the version and then once more with the date which then gets latched).

https://github.com/UbiquityRobotics/ubiquity_motor/blob/5b542a55fd133307150c107d0879a8c42909a598/src/motor_hardware.cc#L152